### PR TITLE
Support Mini Token

### PR DESCRIPTION
**Pull request by forcodedancing**
1. Add following types for mini token
a) MiniTokenIssue
b) MiniTokenSetURI
c) MiniTokenList

2. Tested in local environment and committed the example

3. TODO:
a) Add example once we have testnet ready

### CODE REVIEW

File: src/main/java/com/binance/dex/api/client/TransactionConverter.java [hunk 1]
```diff
@@ -172,13 +178,14 @@ public Transaction convert(byte[] bytes) {
         }
     }
 
+
```
Comment (forcodedancing) [inline src/main/java/com/binance/dex/api/client/TransactionConverter.java:181]: These several changes could be auto formatting. I do not notice this when committing.

File: src/main/java/com/binance/dex/api/client/domain/broadcast/Issue.java [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
-    private String from;
-    private String name;
-    private String symbol;
-    private Long totalSupply;
-    private Boolean mintable;
+    protected String from;
+    protected String name;
+    protected String symbol;
+    protected Long totalSupply;
+    protected Boolean mintable;
```
Comment (fletcher142) [inline src/main/java/com/binance/dex/api/client/domain/broadcast/Issue.java:9]: better remain those as 'private'. if you want to access from sub-class, then can use get** method

File: src/main/java/com/binance/dex/api/client/domain/broadcast/MiniTokenSetURI.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
+package com.binance.dex.api.client.domain.broadcast;
+
+public class MiniTokenSetURI {
+
+    protected String from;
+    protected String symbol;
+    protected String tokenURI;
```
Comment (fletcher142) [inline src/main/java/com/binance/dex/api/client/domain/broadcast/MiniTokenSetURI.java:7]: use private instead

File: src/main/java/com/binance/dex/api/client/domain/broadcast/Issue.java (hunk 2 above)
Comment (forcodedancing) [reply src/main/java/com/binance/dex/api/client/domain/broadcast/Issue.java]: sure. got it.

File: src/main/java/com/binance/dex/api/client/domain/broadcast/MiniTokenSetURI.java (hunk 3 above)
Comment (forcodedancing) [reply src/main/java/com/binance/dex/api/client/domain/broadcast/MiniTokenSetURI.java]: ok, will change this

**Pull request merged**